dashboard/bower.json

```diff
@@ -31,7 +31,6 @@
     "jsonlint": "1.6.0",
     "ng-clip": "0.2.6",
     "ng-lodash": "0.2.3",
-    "ng-password-strength": "0.2.1",
```

Comment:
I have a template for password input fields.
![selection_019](https://user-images.githubusercontent.com/6310786/27278195-453a3efc-54e9-11e7-9870-18553bb04d85.png)

We had the password-strength library which cannot do what we need. That is why I removed it.
FYI: The last release of our password-strength library was in 2015(last commit - on Jan 9, 2016).